src/platform/packages/shared/presentation/presentation_publishing/interfaces/fetch/publishes_pause_fetch.ts

```diff
@@ -13,8 +13,21 @@ export interface PublishesPauseFetch {
   isFetchPaused$: Observable<boolean>;
 }
 
+export interface PublishesEditablePauseFetch extends PublishesPauseFetch {
+  setFetchPaused: (paused: boolean) => void;
+}
```

Comment:
We use this in Discover to pause the fetching of the histogram until the ES|QL variables are ready - see `src/platform/plugins/shared/discover/public/plugin.tsx` and https://github.com/elastic/kibana/pull/242205